### PR TITLE
feat:  storie #52

### PR DESCRIPTION
**Pull request by edsonsnunes**

### CODE REVIEW

File: stories/Tab.stories.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export const TabRight = Template.bind({});
+TabRight.args = {
+  label: 'Right',
+  size: 'right',
+};
+
+export const TabLeft = Template.bind({});
+TabLeft.args = {
+  label: 'Left',
+  size: 'left',
```
Comment (iurynogueira) [inline stories/Tab.stories.ts:58]:
```suggestion
  size: 'left',
```

border?

**Review by iurynogueira — CHANGES_REQUESTED** [review]

File: stories/Tab.stories.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import { Story, Meta } from '@storybook/angular/types-6-0';
+import { TabComponent } from 'projects/ion/src/lib/tab/tab.component';
```
Comment (mizek1) [inline stories/Tab.stories.ts:2]:
```suggestion
import { TabComponent } from '../projects/ion/src/lib/tab/tab.component';
```
This import is breaking the build when you run storybook.

File: stories/Tab.stories.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+export const Basic = Template.bind({});
+Basic.args = {
+  label: 'Custom label',
+};
+
+export const TabSmall = Template.bind({});
+TabSmall.args = {
+  label: 'Small',
+  size: 'sm',
```
Comment (mizek1) [inline stories/Tab.stories.ts:22]:
```suggestion
  tabSize: 'sm',
```
Your args should have the same name that your component's inputs.

Comment (mizek1) [reply stories/Tab.stories.ts:22]: Same for the args below.

File: stories/Tab.stories.ts [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+export const TabLarge = Template.bind({});
+TabLarge.args = {
+  label: 'Large',
+  size: 'lg',
+};
+
+export const TabBottom = Template.bind({});
+TabBottom.args = {
+  label: 'Bottom',
+  size: 'bottom',
```
Comment (mizek1) [inline stories/Tab.stories.ts:40]:
```suggestion
  direction: 'bottom',
```

**Review by mizek1 — CHANGES_REQUESTED** [review]

**Review by iurynogueira — APPROVED** [review]

**Review by Deeved — APPROVED** [review]

**Review by mizek1 — APPROVED** [review]

**Pull request merged**